README.md

```diff
@@ -1,131 +1,231 @@
-# Aurae AE
+<h1 align="center">
+  <code>ae</code>
+  <h3 align="center">UNIX inspired CLI Client for Aurae</h3>
+</h1>
 
-[![Test and Format](https://github.com/aurae-runtime/ae/actions/workflows/001-go-ubuntu-latest-make-test-format-lint.yaml/badge.svg?branch=main)](https://github.com/aurae-runtime/ae/actions/workflows/001-go-ubuntu-latest-make-test-format-lint.yaml)
+<div align='center'>
 
-Unix inspired command line client for Aurae written in Go!
+<a href='https://github.com/aurae-runtime/ae/blob/main/go.mod'>
+<img alt="go version" src="https://img.shields.io/github/go-mod/go-version/aurae-runtime/ae?color=grey&logo=go&style=for-the-badge">
+  
```

Comment:
i'd like to retain the workflow badges please

Reply:
fixed and added in 5b43995.
updated the icons of the others.